WordPress/wporg-showcase-2022
# Blocks: Remove Site Breadcrumbs block code

See https://github.com/WordPress/wporg-mu-plugins/issues/304, https://github.com/WordPress/wporg-mu-plugins/pull/305 — This moves the Site Breadcrumbs block to wporg-mu-plugins, so that we can use it on other sites too.

This PR removes the local `wporg/site-breadcrumbs` block, and since the wporg-mu-plugins one uses the same name, nothing needs to change. The colors on the group block were a little off, so I've fixed that too.

The new block uses `get_bloginfo( 'name', 'display' )` which for Showcase seems to be "The WordPress Showcase," so I'm using the new filter to update that to just "Showcase."

<img width="472" alt="Screenshot 2022-11-28 at 6 46 23 PM" src="https://user-images.githubusercontent.com/541093/204404443-18d96639-1aaf-49f8-b865-0589f2e03d8f.png">

<img width="461" alt="Screenshot 2022-11-28 at 6 48 29 PM" src="https://user-images.githubusercontent.com/541093/204404469-c38ae30a-9a55-4360-94f5-c5aaf55e96f6.png">

To test

- Apply the wporg-mu-plugins PR https://github.com/WordPress/wporg-mu-plugins/pull/305
- Load up this branch
- There should be no visual changes
